README.md

```diff
@@ -50,6 +50,9 @@ const customStyles = {
   }
 };
 
+// Make sure to bind modal to your appElement (http://reactcommunity.org/react-modal/accessibility/)
```

Comment:
Perhaps this can be rephrased like: `In case you want to use aria-hidden, make sure...`.

Reply:
Well, I think if without setting appElement, it will throw warning all the time. So I guess using aria-hidden should be a "must".

Reply:
Hm, ok.